.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+
+jobs:
+  chk_npm_lint:
+    docker:
+      - image: circleci/node
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm run lint
+
+  chk_npm_test:
+    docker:
+      - image: circleci/node
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm run test
+
+workflows:
+  version: 2
+  workflow:
+    jobs:
+      - chk_npm_lint
```

Comment:
It takes 2 minutes to do the code checkout. Wouldn't it be better to run `npm run lint` just as part of the other job?

Reply:
It would be possible, not sure if better. Both have pros and cons. I think it's more convenient to get simultaneous feedback about both linter issues and test failures instead of having to rerun it after fixing one. It also does not affect the overall time because they run in parallel.

But if you mean that it's a bit wasteful - yeah, combining them would definitely use less resources, especially now that I changed it to a `xlarge` machine (though lint still runs in `medium` and might even fit in the smallest one). I thought that it does not matter all that much since there is not much PR traffic in this repo anyway. Should I combine them?

Reply:
The thing is: All CI minutes do cost money :)

Reply:
On the other hand, I think this repo only gets an update once a day, so it might not be that bad after all.

Reply:
I know, but if minutes were the only thing we care about, we'd just move it to Github Actions where we don't pay for them :) And having separate feedback from test and linter means that you can fix both without a rerun in the middle which could save some minutes.

But I'm not really dead set on having them as separate checks. I had a hard time deciding between both options in the first place and I just thought this one is *slightly* nicer. I have just pushed an update that merges the jobs.

So anyway, it would be nice to move off Travis before the next release. Which option do you prefer:
1. Merge this and be done with it but pay for an underutilized `xlarge`.
2. Resurrect #67 and go back to Github Actions.
3. Actually spend some time to optimize the `update` script right now (either by reducing its memory footprint so that we can use a smaller machine or parallelizing it so that `xlarge` gets fully utilized). We'll likely want to do that anyway so it does make sense, I just really wanted to focus on other tasks right now.
4. Stay on Travis until we optimize `update`.

My recommendation would be to go with option 2 but also keep this PR open and switch to CircleCI once we have the script optimized. I'm also fine with staying on Github Actions permanently if you think that's good enough.